src/components/Toggle/ToggleText.jsx

```diff
@@ -1,5 +1,14 @@
+import { ELEMENT_TYPES, getTestId } from "../../utils/test-utils";
+import cx from "classnames";
 import React from "react";
-import { BASE_TOGGLE_CLASS_NAME } from "./ToggleConstants";
+import styles from "./Toggle.module.scss";
 
-const ToggleText = ({ children }) => <span className={`${BASE_TOGGLE_CLASS_NAME}_text`}>{children}</span>;
+const ToggleText = ({ children, id, "data-testid": dataTestId }) => (
+  <span
```

Comment:
This is a dangerous mistake. You removed here the css file but toggle_text css is declared in other file (found it by search). this mistakes can be dangerous and create incidents like we had in the monday-ui-components repo.
Please be more carfeul and make sure there is no other css classes which declared in other file (It will be harder for me to detect all by review)
Before submit a pr to review please make sure you understand if any class that convert by the script is actually in use. if not (like in the text class name) it's a red flag for global class names usage.